src/pages/LoginPage.jsx

```diff
@@ -1,8 +1,55 @@
-import '../assets/styles/index.css';
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import icons from "../assets/svg/svg-icons.svg";
+import "../assets/styles/index.css";
+import "@fontsource/roboto/300.css";
 export const LoginPage = () => {
   return (
-    <>
-     
-    </>
-  )
-}
+    <div className="body_login">
+      <div className="lateral-card">hola mundo</div>
+      <div className="login_area">
+        <div className="card-login">
+          <div className="logo">
+            <svg>
+              <use xlinkHref={`${icons}#svg-users`} />
+            </svg>
+          </div>
+          <Typography align="center" variant="h6" component="h2">
+            Hello Again!
```

Comment:
Please open PRs only with "ready for production" code.

Si es un login page, entonces abrir un PR con la funcionalidad completa.